config/routes.rb

```diff
@@ -67,6 +67,7 @@
   post '/import/:workflow/:format', to: 'import#import'
   get '/import/download_guidance', to: 'import#download_guidance'
 
+  get '/patients/households/self_reporting', to: 'patients#self_reporting'
```

Comment:
Any reason in particular for adding the `household` piece to this route rather than just `/patients/self_reporting`?

Reply:
good question, that's what I originally wanted to do and I was getting `302`'s for a while and then later realized that maybe there was another route such as `/patients/:id` that would be interfering with it. I looked through the routes though and didn't see an existing route exactly like that, but making this change solved the problem

Reply:
Hmm okay. I'd like us to fully understand why so developers don't have to go through the exact same process in the future if it's something we can comment or fix. I can come back to this later and investigate after I go through the other PRs. If you are able to get to the bottom of it in the interim, let me know.

Reply:
While this is a weird issue (and is very likely happening due to what Tim suggested), I actually like the more verbose endpoint name. It may set the stage for future things we add under "households".

Reply:
Yeah, the thing is that I don't even know if that method necessarily _should_ fall under `households` since `self_reporting` includes many people that are not in households at all. Even though it's used to get potential HoHs, but not a big deal either way. I just would like to know exactly why it's occurring. Not really a blocker for this PR to go into 1.14.